src/foam/nanos/auth/UserPropertyAvailabilityService.js

```diff
@@ -45,8 +45,8 @@ foam.CLASS({
 
         Theme theme = (Theme) ((Themes) x.get("themes")).findTheme(x);
         var spid = theme.getSpid();
-        if ( "email".equals(targetProperty) && PreventDuplicateEmailAction.spidGrantsDuplicateEmailPermission(getX(), spid) ) {
-          return true;
+        if ( "email".equals(targetProperty) && PreventDuplicateEmailAction.spidPreventDuplicateEmailPermission(getX(), spid) ) {
+          return false;
```

Comment:
when a spid has 'prevent-duplicate-email' permission it still need to check if there is a user with the email already exists instead of always returning false?